_gsocorgs/2020/cc-in2p3.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Centre de Calcul de l’IN2P3"
+author: "Sebastien Gadrat"
+layout: default
+organization: CC-IN2P3
+logo: logo_ccin2p3.png
```

Comment:
The logo should have the format PROJECT-logo.png

Reply:
:+1: done

Reply:
please do not rename the old logos, but rather copy them since they are used in previous year pages

Reply:
Done